app/controllers/webhooks_controller.rb

```diff
@@ -337,7 +352,7 @@ def send_gift_card_receipt(payment_intent:, amount:, merchant:, receipt_id:)
            '<p> the Send Chinatown Love team</p>' \
            '</body>' \
            '</html>'
-    send_receipt(to: payment_intent.email, html: html)
+    send_receipt(to: payment_intent.recipient.email, html: html)
```

Comment:
LOL omg

Reply:
maybe we can do a logger.info("sending email to ...") in the send_receipt method? 

ideally we'd abstract the emailed receipt sender out into a service or a helper (idk the difference in Rails)